launch/maplab_realsense.launch

```diff
@@ -34,7 +39,26 @@
 
     <param name="infrared/enabled" value="true"/>
     <param name="infrared/subsample_factor" value="1"/>
+    <param name="infrared/enable_auto_exposure" value="true"/>
+    <param name="infrared/emitter_enabled" value="true"/>
+    <!-- Values for outdoor calibration, disable autoexposure
+         for these settings to take effect. -->
+    <param name="infrared/gain" value="1"/>
+    <param name="infrared/exposure" value="5"/>
 
+<<<<<<< HEAD
```

Comment:
git conflict

Reply:
Looks like github is lagging again, I fixed it and pushed it without this conflict..